commands/help.rb

```diff
@@ -131,8 +131,6 @@ def self.help(command)
         Update crew.
         Usage: crew update
         This only updates crew itself.  Use `crew upgrade` to update packages.
-        Usage: crew update compatible
-        This updates the crew package compatibility list.
```

Comment:
I'm not sure of the ramifications of removing this.

Reply:
None. The last remaining usage was in `crew whatprovides`, and I've already removed that. We just use `PackageUtils.compatible?` instead.

Reply:
Should we purge it from device.json too?

Reply:
We could, its not doing any harm by being there and it won't show up on new installs. I don't really mind either way, so I didn't implement it.